tests/Makefile

```diff
@@ -0,0 +1,25 @@
+NAME = gameboy_test
+
+OBJ = cpu_tests.o ../Z80_CPU.o
+
+SOURCE = cpu_tests.cpp ../Z80_CPU.cpp
+
+FLAGS = -I $(HOME)/shared/ -L $(HOME)/shared/gtest/ -lgtest_main -lgtest -std=c++11
+
+all : $(NAME)
+
+$(NAME) : $(OBJ) $(HEADERS)
+	g++ $(FLAGS) $(OBJ) -o $(NAME)
+
+%.o : %.cpp $(HEADERS)
+	g++ -I $(HOME)/shared/ -std=c++11 -c $< -o $@
```

Comment:
Why you didn't call $(FLAGS) here also ?

Reply:
Because this is the compiling stage i only need the source of the header files specified with the `-I`, the $(FLAGS) contain information the the linking stage require.

Reply:
You can make two FLAGS and include one in the other, that way it will be simple to edit these flags in the future